packages/path_provider/path_provider/example/integration_test/path_provider_test.dart

```diff
@@ -6,8 +6,9 @@
 
 import 'dart:io';
 import 'package:flutter_test/flutter_test.dart';
-import 'package:path_provider/path_provider.dart';
 import 'package:integration_test/integration_test.dart';
+import 'package:path_provider/path_provider.dart';
+
```

Comment:
looks like the formatter is unhappy with some of the extra lines here?

Reply:
Just seeing that.  Looking now.